README.md

```diff
@@ -1 +1,17 @@
-# pokedex-2018-honeybadgers
\ No newline at end of file
+# pokedex-2018-honeybadgers
+
+### Database Setup
+
+Step 1: Create database
+
+Using the database manager of your choice, create a database called 'pokedex_hb'.
+
+Step 2: Create table
+
+Then run 'sql/pokedex_2018-11-12.sql' in the database manager.  
+This will create the pokemon table.
+
+Step 3: Populate pokemon table
+
+Open your terminal and navigate to 'pokedex-2018-honeybadgers' and run 'php -f dataScrape/apiPull.php'.  
```

Comment:
If your vagrantbox works this part doesn't. You need to go into pokemonTableUpdate.php and manually change the IP address to the IP address of your local server.

Reply:
Changed in ed03b9f.